automatic/jenkins/update.ps1

```diff
@@ -15,16 +15,13 @@ function global:au_SearchReplace {
 }
 
 function global:au_GetLatest {
-    # needed for Invoke-WebRequest to work for this site
-    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-
     $page = Invoke-WebRequest -Uri $releases -UseBasicParsing
-    $regexUrl = "Deploy Jenkins`n(?<version>[\d\.]+)"
-    $page.RawContent -match $regexUrl
-    $version = $matches.version
+    $regex  = 'jenkins-.+\.zip$'
```

Comment:
Can you change the code to match the format that is above. There are a myriad of ways to do this but I prefer the way I have it as it's much simpler to read.